src/main/scala/mesosphere/marathon/event/http/HttpEventActor.scala

```diff
@@ -2,53 +2,91 @@ package mesosphere.marathon.event.http
 
 import akka.actor._
 import akka.pattern.ask
+import mesosphere.marathon.api.v2.json.Formats._
 import mesosphere.marathon.event._
+import mesosphere.marathon.event.http.HttpEventActor._
 import mesosphere.marathon.event.http.SubscribersKeeperActor.GetSubscribers
-import play.api.libs.json.{ JsValue, Json }
+import mesosphere.marathon.metrics.Metrics
 import spray.client.pipelining.{ sendReceive, _ }
 import spray.http.{ HttpRequest, HttpResponse }
 import spray.httpx.PlayJsonSupport
-import mesosphere.marathon.api.v2.json.Formats._
 
 import scala.concurrent.Future
-import scala.util.{ Failure, Success }
+import scala.concurrent.duration._
+import scala.util.Success
+import System.currentTimeMillis
+
+object HttpEventActor {
```

Comment:
Not the fault of this PR but some documentation about what this actor does and what the implementation goals are would be nice.